src/field/index.js

```diff
@@ -137,6 +137,10 @@ export function getFieldLabel(m, field, notFound = '') {
   return getField(m, field).get('label', notFound);
 }
 
+export function getFieldHint(m, field, hint, notFound = '') {
```

Comment:
getFieldPlaceholder  is more suitable.
nit: hint -> property

Reply:
Done